collection-scripts/gather

```diff
@@ -1,24 +1,66 @@
 #!/bin/bash
+# shellcheck disable=SC2154,SC1091,SC2086
+source common.sh
 
-# generate /must-gather/version file
-. version
-echo "opendatahub/must-gather" > /must-gather/version
-version >> /must-gather/version
+# for new flag to get since and since-time
+get_log_collection_args
 
+
+# generate /must-gather/version file with RHOAI version
+echo "red-hat-data-services/must-gather" > /must-gather/version
+rhoai_version >> /must-gather/version
```

Comment:
Should it be "$DST_DIR/version"?

Reply:
and may be `[[ -d $DST_DIR ]] || mkdir -p $DST_DIR`

Reply:
yes, should use variable than hardcoded here